sql/views/pgbouncer_fdw_views.sql

```diff
@@ -33,7 +34,8 @@ CREATE VIEW @extschema@.pgbouncer_clients AS
         , ptr text
         , link text
         , remote_pid int
-        , tls text);
+        , tls text
+        , application_name text);
```

Comment:
```suggestion
        , application_name text
        );
```
? Just to make further columns just single-line diffs, since you're already using leading comma notation. (Don't care strongly, so feel free to disregard...)

Reply:
Good idea. Went back and did it for all views